forms-flow-admin/src/components/users/users.tsx

```diff
@@ -352,12 +354,16 @@ const Users = React.memo((props: any) => {
 
           <div className="user-filter-container  col-lg-4 col-xl-4 col-md-4 col-sm-6 col-12 d-flex justify-content-end gap-2">
             <span className="my-2">{t("Filter By:")} </span>
-            <Form.Select className="bg-light text-dark" onChange={handleSelectFilter} title={t("Filter here")}>
-              <option value="ALL" selected={!props.filter}>
+            <Form.Select 
+             className="bg-light text-dark"
+             onChange={handleSelectFilter} 
+             title={t("Filter here")} 
+             data-testid="users-roles-filter-select">
+              <option value="ALL" selected={!props.filter} data-testid="users-all-roles-option">
```

Comment:
the options need an uniformity in data testid 
example: `users-roles-filter-option-all ` for all users
                `users-roles-filter-option-${variable}` for other option

Reply:
updated